projects/graph/src/graph.py

```diff
@@ -52,3 +52,26 @@ def traverse_breath_first(self, start_id):
                 open_vertices.put(linked_id)
         # Print path
         print(F'Path: {path}')
+
+    def traverse_depth_first_stack(self, start_id):
+        """Prints a path of vertices visited, starting at a given vertex."""
+        path = []
+        # Define the open and closed vertices trackers
+        open_vertices = []
+        closed_vertices = set()
```

Comment:
Great implementation with stack as well as comments, your process is clear and it looks like you have a good hold on DFS. Keep pushing forward!